packages/base64/CHANGELOG.md

```diff
@@ -3,6 +3,24 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+# 0.2.0 (2021-05-28)
+
+
+### Bug Fixes
+
+* update to eslint 7.23.0 ([#652](https://github.com/endojs/endo/issues/652)) ([e9199f4](https://github.com/endojs/endo/commit/e9199f41c511b5df10593d931febdd90693b011a))
+* **base64:** Fix minor docs typo in usage ([185234e](https://github.com/endojs/endo/commit/185234efd86673e647d9ef303a5626233ac659dd))
+* **base64:** Refine package.json ([fea29e5](https://github.com/endojs/endo/commit/fea29e5e3ffb574b32cf09bf0039f0816a203511))
+
+
+### Features
+
+* **base64:** Export package.json ([c5a0e53](https://github.com/endojs/endo/commit/c5a0e534cfb43c32ae3eb62c3bd85a8652f8b417))
```

Comment:
Based on these conventional commits, I expected a patch bump. There are no breaking changes.

Reply:
New feature means to bump the minor version.